test/regress_http.c

```diff
@@ -1643,9 +1643,6 @@ http_fill_backlog(struct event_base *base, int port)
 		evcon[i] = NULL;
 
 		return evcon;
- end:
```

Comment:
Have you even tried to compile after you changes? `tt_assert`/`tt_int_op` uses `goto end`